main.go

```diff
@@ -13,9 +12,13 @@ func main() {
 		log.Fatalf("Error loading config: %s", err.Error())
 	}
 
-	var fileData = playlist_creator.ReadFiles(configData)
+	err, fileData := playlist_creator.ReadFiles(configData)
+	if err == nil {
```

Comment:
the convention is to return the error as the last value, e.g.
```
a,b,c, err := doSomething(…)
```
and also a typo maybe.

```suggestion
	fileData, err := playlist_creator.ReadFiles(configData)
	if err != nil {
```